mne/viz/misc.py

```diff
@@ -578,3 +579,92 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
     if show:
         fig.show()
     return fig
+
+
+def plot_phase_amplitude_coupling(phase_bins, normalized_amplitude,
+                                  title=None, show=True):
+    """Plot the phase amplitude plot.
+
+    Parameters
+    ----------
+    phase_bins : array, shape (number of bins + 1,)
+        Binned phase time series of phase modulating signal.
+    normalized_amplitude : ndarray, shape (n_epochs, bin_num)
+        The normalized amplitude values across each bin.
+    title : str
+        Title, default is 'Phase amplitude plot'.
+    show : bool
+        Call pyplot.show() at the end. Default is True.
+
+    Returns
+    -------
+    fig : Instance of matplotlib.figure.Figure
+          Figure
+    """
+    import matplotlib.pyplot as plt
+    title = 'Phase amplitude plot' if title is None else title
+    for i in range(len(phase_bins) - 1):  # the bins are always more
+        plt.locator_params(axis='x', nbins=4)
+        plt.bar(np.rad2deg(phase_bins[i]), normalized_amplitude[i],
+                width=10, align='edge')
+        plt.xlabel('Phase bins (deg)')
+        plt.ylabel('Normalized Mean Amplitude')
+        plt.title(title)
+    plt.tight_layout()
+
+    if show:
+        plt.show()
+    return plt.gcf()
+
+
+def plot_cross_frequency_coupling(times, freqs, traces, ztraces,
+                                  z_threshold, avg, show=True):
+    """Plot cross frequency coupling.
+
+    Parameters
+    ----------
+    times : array, shape (n_times,)
+        Time points for signal plotting.
+    freqs : array, shape (n_freqs,)
+        Frequency points across range at which amplitudes are computed.
+    traces : ndarray, shape (n_epochs, n_times)
+        Normalized amplitude traces.
+    ztraces : ndarray, shape (n_epochs, n_times)
+        Statistically significant amplitude traces.
+    z_threshold : float
+        Threshold of statistically significant amplitude traces.
+    avg : array, shape (n_times,)
+        Average (or evoked) signal.
+    show : bool
+        Call pyplot.show() at the end. Default True.
+
+    Returns
+    -------
+    fig : Instance of matplotlib.figure.Figure
+          Figure
+
+    """
+    import matplotlib.pyplot as plt
+
+    fig = plt.figure()
+    ax1 = plt.subplot2grid((3, 1), (0, 0), rowspan=2)
+    ax2 = plt.subplot2grid((3, 1), (2, 0), rowspan=1)
+    vmax = np.max(np.abs(traces))
+    vmin = -vmax
+    traces_plot = np.ma.masked_array(traces, np.abs(ztraces) < z_threshold)
+
+    ax1.pcolor(times, freqs, traces, vmin=vmin, vmax=vmax,
+               cmap=plt.cm.gray)
+    ax1.pcolor(times, freqs, traces_plot, vmin=vmin, vmax=vmax,
+               cmap=plt.cm.jet)
+    ax1.axis('tight')
+    ax1.set_ylabel('Freq (Hz)')
+
+    ax2.plot(times, avg, 'k')
+    ax2.set_ylim([np.min(avg), np.max(avg)])
+    ax2.set_xlabel('Times (s)')
+    ax2.set_ylabel('Average')
+    tight_layout()
+    if show:
+        plt.show()
+    return fig
```

Comment:
Hrm - so those plotting functions are meant to be called at the end of the analysis pipeline for this data. They are called in the examples but as you can see, it takes a few seconds for them to run. Is that too much for tests? I could try to reduce the number of permutations and just make sure it doesn't error.

Either way, going through the tests now I don't think that they cover very much of these functions. There isn't actually a test to call the "main" functions that have been added. I will try to write these in.